3-enrich/spark-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.spark/EnrichJobSpec.scala

```diff
@@ -242,6 +243,16 @@ object EnrichJobSpec {
                 |"uri": "http://iglucentral.com"
               |}
             |}
+          |},
+          |{
+            |"name": "Iglu Central YAUAA enrichment",
+            |"priority": 1,
+            |"vendorPrefixes": [ "com.snowplowanalytics" ],
+            |"connection": {
+              |"http": {
+                |"uri": "http://iglucentral-dev.com.s3-website-us-east-1.amazonaws.com/yauaa_enrichment"
+              |}
+            |}
```

Comment:
note to remove once the iglu central pr is merged

Reply:
Ok so this can't be merged until the PR on Iglu is merged.

Reply:
yup